StreamChatUITestsAppUITests/MockServer/MessageDetails.swift

```diff
@@ -0,0 +1,47 @@
+//
+// Copyright © 2022 Stream.io Inc. All rights reserved.
+//
+
+@testable import StreamChat
+import XCTest
+
+extension StreamMockServer {
+    
+    func saveMessageDetails(
```

Comment:
Not sure what's going on here, but I think this file (`MessageDetails.swift`) is out of date. I can't even find it in the source code. `MessageList.swift` is the right one.